Assets/_Functionalities/ObjectInformation/Scripts/BagInspector.cs

```diff
@@ -404,6 +404,8 @@ private void LoadFeatureContent(FeatureMapping mapping)
 			Dictionary<string, object> properties = mapping.Feature.Properties as Dictionary<string, object>;
 			foreach (KeyValuePair<string, object> property in properties)
 			{
+				if(property.Value == null) continue;
```

Comment:
Is this the real fix and the rest refactoring?

Reply:
no, the real fix is that the optical raycaster function was being used incorrectly and returning (0,0,0), @bozmir can explain what this does

Reply:
could be that the opticaly raycaster had impact on this, but from what i saw was that a nullref exception was given for only some values in dictionairy preventing the rest of the loop from executing and populating the ui